homeassistant/components/bluetooth/scanner.py

```diff
@@ -168,22 +169,6 @@ async def async_diagnostics(self) -> dict[str, Any]:
             "start_time": self._start_time,
         }
 
-    @hass_callback
-    def async_register_callback(
```

Comment:
This is not a breaking change?

Reply:
This is not a breaking change because it's only used internally to the Bluetooth integration and this object is never exposed externally

Reply:
Ok.

Reply:
Thanks!